ui/src/app/pages/profile/data-use-agreement-content.tsx

```diff
@@ -0,0 +1,200 @@
+import {BolderHeader, BoldHeader} from 'app/components/headers';
+import * as React from 'react';
+import {IndentedListItem, IndentedOrderedList, IndentedUnorderedList, SecondHeader} from './data-use-agreement';
+
+const CORE_VALUES_URL = 'https://allofus.nih.gov/about/about-all-us-research-program';
+
+const AoUTitle = () => {
+  return <span><i>All of Us</i> Research Program</span>;
+};
+
+{/* NOTE: Make sure to update dataUseAgreementVersion if there is any change to the DUA text. */}
+export const DataUseAgreementContent = () => {
+  return <div>
+    <BolderHeader style={{display: 'flex', justifyContent: 'center'}}><AoUTitle/></BolderHeader>
+    <BoldHeader  style={{display: 'flex', justifyContent: 'center'}}>Demonstration Project Data Use Agreement</BoldHeader>
+    <p>
+      This data use agreement describes how <AoUTitle/> data can and cannot be used for the purposes
+      of program-approved demonstration projects.
+    </p>
+    <p>
+      This is an agreement between Vanderbilt University Medical Center and authorized demonstration users of data
+      from the <AoUTitle/>.
+    </p>
+    <p>
+      An <strong>authorized demonstration user</strong> is a person who is authorized to access
+      and/or work with registered or
+      controlled tier data from the <AoUTitle/> for the exclusive purpose of a program-approved
+      demonstration project. Authorized demonstration users are limited to trainees, faculty or staff at <AoUTitle/>
+      consortium partner institutions.
```

Comment:
There's a missing space between _All of Us Research Program_ and consortum here. (I think it works if you breakline before the `at <AoUTitle/>` so it counts whitespace. It trims before and after newlines when there are new tags.)

Reply:
that's good to know, thanks.